libraries/botbuilder-dialogs-declarative/package.json

```diff
@@ -28,20 +28,22 @@
     }
   },
   "dependencies": {
+    "bf-cli": "^1.0.4",
```

Comment:
Is the `bf-cli` tool used at runtime, or just when testing?

Reply:
Just testing. Should I remove it from here and then just handle manual installation? The test kind of does that.